### ISSUE TITLE
Alphabetically sort grouped namespaces fix breaks code

### ISSUE THREAD

**Issue by ainehickey** [slevomat/coding-standard#872]
Sniff: SlevomatCodingStandard.Namespaces.AlphabeticallySortedUses
Version: slevomat/coding-standard - 6.1.1, squizlabs/php_codesniffer - 3.5.3

I think I might have found a bug.
I have a file. The Namespaces aren't alphabetic, but I'm using grouped namespaces.

**Before fix code:**
```php
<?php

declare(strict_types=1);

use Illuminate\Database\Seeder;
use App\Models\Entities\{
    A,
    B,
    C
};
use Laravel\Lumen\Testing\DatabaseTransactions;

class Example
{
}
```

I'd like my namespaces to be alphabetic.

**My ruleset:**
```xml
<?xml version="1.0"?>
<ruleset name="MyRuleset">
    <config name="installed_paths" value="vendor/slevomat/coding-standard" />
    <rule ref="SlevomatCodingStandard.Namespaces.AlphabeticallySortedUses" />
</ruleset>
```

When I run phpcs it over my file, I get an error. Super, as this is correct
```
-------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-------------------------------------------------------------------------------------------------------------------------------------
 15 | ERROR | [x] Use statements should be sorted alphabetically. The first wrong one is Laravel\Lumen\Testing\DatabaseTransactions.
-------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-------------------------------------------------------------------------------------------------------------------------------------
```

But when I try to automatically fix it, I run into an issue.

```
PHPCBF RESULT SUMMARY
------------------------------------------------------------------------------------------
FILE                                                                      FIXED  REMAINING
------------------------------------------------------------------------------------------
C:\Example.php                                                        1      0
------------------------------------------------------------------------------------------
A TOTAL OF 1 ERROR WERE FIXED IN 1 FILE
------------------------------------------------------------------------------------------
```

**After fix code:**
```php
declare(strict_types=1);

use A;
use Illuminate\Database\Seeder;
use Laravel\Lumen\Testing\DatabaseTransactions;
```

The grouped namespace outside the brackets has been deleted, and one one of the class names remains.

Please let me know if I'm doing something wrong or if you need any more info.

**Comment by kukulich** [comment]
Yes, we don’t support group uses.

**Comment by ainehickey** [comment]
> Yes, we don’t support group uses.

Ah, ok. Thanks.
